### PR TITLE
[Docs][Connector-V2][StarRocks]Reconstruct the StarRocks connector document

### PR DESCRIPTION
**Pull request by TaoZex**
## Purpose of this pull request

subtask of https://github.com/apache/seatunnel/issues/4572
Reconstruct the StarRocks connector document

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by TyrantLucifer** [comment]
@Hisoka-X @EricJoy2048 PTAL

**Comment by davidzollo** [comment]
good job

**Comment by zhilinli123** [comment]
Add a connector version for this
 https://github.com/apache/seatunnel/issues/4572 @TaoZex thks

**Review by hailin0 — DISMISSED** [review]

**Comment by liugddx** [comment]
> Add a connector version for this #4572 @TaoZex thks

+1

File: docs/en/connector-v2/sink/StarRocks.md [hunk 1]
```diff
@@ -2,94 +2,44 @@
...
+| password                     | string  | yes      | -               | `StarRocks` user password                                                                                                                                                                                                    |
+| database                     | string  | yes      | -               | The name of StarRocks database                                                                                                                                                                                               |
+| table                        | string  | no       | -               | The name of StarRocks table, If not set, the table name will be the name of the upstream table                                                                                                                               |
+| labelPrefix                  | string  | no       | -               | The prefix of StarRocks stream load label                                                                                                                                                                                    |
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/StarRocks.md]: From Doris Sink Connector I found use stream load label can implement 2PC and make sink support `exactly-once`. What does this look like in Starlocks?

File: docs/en/connector-v2/sink/StarRocks.md [hunk 2]
```diff
@@ -158,12 +161,29 @@ sink {
     }
   }
 }
-
 ```
 
-Use CSV format to import data
+### Support write cdc changelog event(INSERT/UPDATE/DELETE)
```
Comment (EricJoy2048) [inline docs/en/connector-v2/sink/StarRocks.md:166]: Can you add an example for `save_mode_create_template ` option?

File: docs/en/connector-v2/sink/StarRocks.md (hunk 1 above)
Comment (TaoZex) [reply docs/en/connector-v2/sink/StarRocks.md]: Currently, starrocks does not support exactly-once

Comment (EricJoy2048) [reply docs/en/connector-v2/sink/StarRocks.md]: Ok

**Review by davidzollo — DISMISSED** [review]

**Review by ic4y — APPROVED** [review]

**Review by liugddx — APPROVED** [review]
+1

**Pull request merged**